quill-core/src/main/scala/io/getquill/util/Messages.scala

```diff
@@ -4,6 +4,11 @@ import scala.reflect.macros.blackbox.{ Context => MacroContext }
 
 object Messages {
 
+  private val debugEnabled =
```

Comment:
yes this is intentional, I mean our CI system or a clean compile produces over >10k lines on our systems (already).